be/src/exprs/table_function/table_function_factory.cpp

```diff
@@ -91,6 +91,7 @@ TableFunctionResolver::TableFunctionResolver() {
     add_function_mapping("unnest", {TYPE_ARRAY}, {TYPE_DATETIME}, func_unnest);
     add_function_mapping("unnest", {TYPE_ARRAY}, {TYPE_BOOLEAN}, func_unnest);
     add_function_mapping("unnest", {TYPE_ARRAY}, {TYPE_ARRAY}, func_unnest);
+    add_function_mapping("unnest", {TYPE_ARRAY}, {TYPE_STRUCT}, func_unnest);
```

Comment:
Why don't you support all types, like MAP, STRUCT, JSON?

Reply:
there is a customer need this feature, just open struct first. also i think we can open map/json too, will do it

Reply:
Done